core/src/test/scala/unit/kafka/server/DynamicConfigChangeTest.scala

```diff
@@ -472,6 +472,80 @@ class DynamicConfigChangeTest extends KafkaServerTestHarness {
     }
   }
 
+  private def alterBrokerConfigs(brokerId: String, newValue: java.lang.Long): Unit = {
+    if (isKRaftTest()) {
+      val admin = createAdminClient()
+      try {
+        val resource = new ConfigResource(ConfigResource.Type.BROKER, brokerId)
+        val configEntry = new ConfigEntry(QuotaConfigs.LEADER_REPLICATION_THROTTLED_RATE_CONFIG, newValue.toString)
+        val configEntry2 = new ConfigEntry(QuotaConfigs.FOLLOWER_REPLICATION_THROTTLED_RATE_CONFIG, newValue.toString)
+        val configEntry3 = new ConfigEntry(QuotaConfigs.REPLICA_ALTER_LOG_DIRS_IO_MAX_BYTES_PER_SECOND_CONFIG, newValue.toString)
+        val config = new Config(List(configEntry, configEntry2, configEntry3).asJavaCollection)
+        admin.alterConfigs(Map(
```

Comment:
Could you please use `incrementalAlterConfigs` instead of deprecated  `alterConfigs`?